yarn.lock

```diff
@@ -6663,6 +6663,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@xata.io/client@npm:^0.25.1":
```

Comment:
Revert along with `package.json` change above

Reply:
I removed it from package.json, however if I remove this block it is added back by `yarn install`. Could it because of the reference to it in the example folder?